store/sql_channel_store.go

```diff
@@ -1636,6 +1639,43 @@ func (s SqlChannelStore) performSearch(searchQuery string, term string, paramete
 	return result
 }
 
+func (s SqlChannelStore) GetAllDefaultChannels() StoreChannel {
```

Comment:
I like the idea of all this, but it shouldn't be necessary any more since we have the channel in CreatePost and don't need to go to the DB to get it again. Plus, it will be missing any teams and channels that are created after the server has been started up